Resources/public/js/components/products/components/edit/media/main.js

```diff
@@ -47,8 +59,7 @@ define(['config', 'suluproduct/util/product-delete-dialog'], function(Config, De
 
         bindCustomEvents: function() {
             this.sandbox.on('product.state.change', function(status) {
-                if (!this.options.data ||
-                    !this.options.data.attributes.status ||
+                if (!this.options.data || !this.options.data.attributes.status ||
```

Comment:
`||` into next line and if one will break down then all terms should go into a new line